tests/tutorials/test_sanity_tutorials.py

```diff
@@ -55,7 +55,8 @@
              'unsupervised_learning/index.md',
              'vision/index.md',
              'tensorrt/index.md',
-             'tensorrt/inference_with_trt.md']
+             'tensorrt/inference_with_trt.md',
```

Comment:
can you rebase or remove from this tutorial?

Reply:
I didn't quite get what you meant here.

Reply:
I was wondering if this has already been fixed and you just need to rebase

Reply:
This might have been fixed, but I am not modifying the tensorrt/inference_with_trt.md tutorial here. The only thing I have done is to adda ```,``` after it and in the new line whitelisted the java tut.

Reply:
And I tried rebasing with the master branch, it seems like java-api branch itself is far behind master right now, so if I base my branch from the master, I'll be adding tons of commits here.